index.js

```diff
@@ -4,18 +4,21 @@ let Promise = require('bluebird')
 let readDir = Promise.promisify(require('recursive-readdir'))
 let fs = Promise.promisifyAll(require('fs'))
 
+let slice = start => thing => thing.slice(start)
+
 // Path Utils
-let filesRelative = (dir, ex) => readDir(dir, ex).map(x => x.slice(dir.length))
+let relativeFilenames = (dir, exclusions) => readDir(dir, exclusions).map(n => slice(dir.length)(n))
```

Comment:
Oh, I guess I didn't know exactly what you meant by point-free. I just read this: https://wiki.haskell.org/Pointfree and my head exploded. :(